geodata_mart/templates/maps/map.html

```diff
@@ -456,14 +542,57 @@ <h5 class="modal-title" id="bufferInfoModalLabel">{% translate "Buffer Configura
       L.Toolbar2.EditAction.Popup.Edit,
     ]
 
-  L.tileLayer("https://tile.openstreetmap.org/{z}/{x}/{y}.png", {
-    attribution: 'Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>.',
-  }).addTo(map)
+  // Info button click event handler
+    var infoContent = `
+      <div class="info-popup">
+        <div class="info-content">
+          <h3>Clip and ship Quick Help</h3>
+          <p>Select a couple of map layers from the map tab.</p>
```

Comment:
@seabilwe Can you suggest some quick help text here, If we have a full tutorial of how this works we could also add reference to a URL here

Reply:
@NyakudyaA This video https://youtu.be/SD3IF7xkeJU was recorded last year to guide users on how to navigate the platform. 

After the editions you've made, it would have to be re-recorded again. For now, you can add this as a placeholder.